tests/unit/Http/RouteCollectionTest.php

```diff
@@ -44,4 +44,17 @@ public function can_add_routes_late()
 
         $this->assertEquals('/posts', $routeCollection->getPath('forum.posts.delete'));
     }
+
+    /** @test */
+    public function can_add_empty_parameters()
```

Comment:
I think it's more correct to say can't, since it properly errors out now for required parameters.

```suggestion
    public function must_provide_required_parameters()
```